core/src/main/java/com/amplifyframework/predictions/models/TextFormatType.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Specifies the type of text detection to perform.
+ */
+@SuppressWarnings("JavadocVariable")
+public enum TextFormatType implements IdentifyAction {
+    FORM,
+    TABLE,
+    PLAIN,
+    ALL;
```

Comment:
We should get JavaDocs on these, indicating why a user would choose to select one of them. What are the implications of choosing `FORM` vs. `TABLE`, vs. `PLAIN`? What does `ALL` mean - it does all of the first three? What are the result outputs if I select each one of these - do they differ?